art-cluster/pipelines/data/project/art-cd/image/files/krb5-redhat.conf

```diff
@@ -0,0 +1,32 @@
+[libdefaults]
```

Comment:
We have several copies of this same file at:
- ./doozer/.devcontainer/krb5-redhat.conf
- ./elliott/.devcontainer/krb5-redhat.conf
- ./.devcontainer/krb5-redhat.conf

and there's only a couple of differences from this one. Maybe worth unifying?

Reply:
Changed in https://github.com/openshift-eng/art-tools/pull/171/commits/d8ab9eaf9a928c2e83679881ec2fb75f4336b323

Reply:
Created a separate PR https://github.com/openshift-eng/art-tools/pull/490